module/VuFind/src/VuFind/ILS/Driver/MultiBackend.php

```diff
@@ -1417,6 +1417,9 @@ protected function callMethodIfSupported(
         bool $stripPrefixes = true,
         bool $addPrefixes = true
     ) {
+        if (empty($params) && null === $source) {
```

Comment:
I think with MultiBackend we could have a slightly different logic: return true if there are no configured login targets ( = getLoginDrivers returns an empty array). MultiBackend could also check all configured login targets for loginIsHidden support and verify that at least one of the configured login targets allows login, but I wouldn't go that far without a use case.

In a related note, I'm not quite sure if loginIsHidden works with ChoiceAuth at all, but that's a different issue.

Reply:
@demiankatz I understand that if ILS is used as the login method, but I can't see the point with ChoiceAuth, Shibboleth, Database or any other one. Why shouldn't the user be able to e.g. log in with Shibboleth to access electronic material when the ILS is down?

Reply:
@demiankatz So the problem is that the new MultiBackend driver tries to handle it but fails since it can't determine the backend at least unless you define a default_driver in MultiBackend.ini.

Reply:
Perhaps the best solution for this particular functionality is to default to false if no backend can be determined; I can't imagine there's a particularly good reason to dynamically hide login in a multi-backend scenario.

Reply:
Indeed. And the same applies to a couple of other methods as well that should be disabled when default_driver is not set.

Reply:
I've now opened #2174 that should cover all the methods that don't have source-specific parameters.